### ISSUE TITLE
Cloning To Windows Gives "Failed to load resource: net::ERR_CONNECTION_REFUSED"

### ISSUE THREAD

**Issue by austincollinpena** [Niraj-Fonseka/grpc_stream-medium#2]
Process:

1. Start up Envoy
2. Run the Go server
3. Run the React app

Error code: Failed to load resource: net::ERR_CONNECTION_REFUSED

Any ideas?

**Comment by kumaranshu72** [comment]
Most probably the connection between envoy and client is failing

**Comment by delaprada** [comment]
I have the same problem. Did you solve it?

**Comment by austincollinpena** [comment]
I did not

**Comment by Niraj-Fonseka** [comment]
Sorry guys just seeing this let me see if I can replicate it tonight / tomorrow and will post the findings here

**Comment by kumaranshu72** [comment]
@austincollinpena It will be helpful if you can share the configurations.

**Comment by austincollinpena** [comment]
I ended up abandoning the project, apologies but you'll have to get someone else who has the same issue to help

**Comment by Niraj-Fonseka** [comment]
Yeah I'm not able to replicate this. Will you be able to provide a more descriptive error message ? Maybe a screen shot or something ?

**Comment by fmr999** [comment]
Hi I'm getting the same issue when I try to replicate,
I get from the browser
```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://localhost:8000/sensors.Sensor/TempSensor. (Reason: CORS request did not succeed).
```

**Comment by kumaranshu72** [comment]
@fmr999 Set the cors headers in envoy.
You can refer to the config here for more details.
https://stackoverflow.com/a/62905330/7494233
